gswidgetkit/number_field.py

```diff
@@ -60,13 +62,17 @@ def __init__(self, parent, id=wx.ID_ANY, label="", default_value=0, min_value=0,
         else:
             self.scroll_dir = 1
 
+        self.type_ = type_
         self.cur_value = default_value
         self.min_value = min_value
         self.max_value = max_value
         self.change_rate = .5
         self.change_value = 0
+        self.step_size = step_size
         self.suffix = suffix
-        self.value_range = [i for i in range(min_value, max_value)]
+
+        self.value_range = [min_value, max_value]
```

Comment:
I changed this for two reasons:
1. Nothing broke
2. Performance reasons i.e. when there is a large value range